test/test_playbooks/tasks/repository.yml

```diff
@@ -5,6 +5,7 @@
     - product_name: "Test Product"
     - repository_name: "Test Repository"
     - repository_content_type: "yum"
+    - repository_label: "just a test repo"
```

Comment:
I don't think labels can have spaces.

Reply:
fixed in https://github.com/theforeman/foreman-ansible-modules/pull/160/commits/7f50f090d0766d842a4366303dc906d96694ad52